app/src/androidTest/java/ch/epfl/sdp/peakar/TestingConstants.java

```diff
@@ -50,4 +50,5 @@
 
     //Cache constants
     public static final String CACHE_FILE_NAME_TEST = "POIPointsCache.txt";
+    public static final String CACHE_FILE_PATH_TEST = "";
```

Comment:
When is this used? Do we need an empty constant? Or can this be refactored to not be used at all?

Reply:
You are right, that is some rest from my intense debug session